plugins/org/src/components/Cards/Group/MembersList/MembersListCard.tsx

```diff
@@ -106,14 +107,14 @@ const MemberComponent = (props: { member: UserEntity }) => {
             }}
             textAlign="center"
           >
-            <Typography variant="h5">
+            <Typography variant="h6">
```

Comment:
I changed this to an h6 because of the implementation of `OverflowTooltip`. The component uses [TextTruncate](https://github.com/backstage/backstage/blob/master/packages/core-components/src/components/OverflowTooltip/OverflowTooltip.tsx#L59) from `react-text-truncate`, which truncates based on the last line specified. In other words, if you add in `line={2}` this will then truncate the second line, and if you leave it blank it'll truncate the first line. So, the current implementation means all names will now be on a single line and I reduced the fontSize to have it fit better. If we do want names to be continued to have 2 lines, we'd have to modify this implementation because otherwise you get some funky results when it's one long name or the first name is longer than the last name, see below. I can look into this more if we're worried about names being limited to one line! 

<img width="371" alt="image" src="https://user-images.githubusercontent.com/72715376/219494222-16b83ef3-ffec-42a4-b4d7-9e9455f1de17.png">